tensorboard/webapp/webapp_data_source/tb_feature_flag_data_source.ts

```diff
@@ -29,14 +29,13 @@ import {
 } from './tb_feature_flag_data_source_types';
 
 const DARK_MODE_MEDIA_QUERY = '(prefers-color-scheme: dark)';
+const FEATURE_FLAG_STORAGE_KEY = 'tb_feature_flag_storage_key';
 
 // TODO(tensorboard-team): QueryParamsFeatureFlagDataSource now is a misnomer as
 // it also sources the data from media query as well as the query parameter.
 // Decide how to move forward with more sources of the data + composability.
 @Injectable()
-export class QueryParamsFeatureFlagDataSource
-  implements TBFeatureFlagDataSource
-{
+export class FeatureFlagOverrideDataSource implements TBFeatureFlagDataSource {
```

Comment:
Note that renaming the class (which is the right thing to do) is going to cause problems importing into the code base. Easiest way to work around it is to have:
`export QueryParamsFeatureFlagDataSource = FeatureFlagOverrideDataSource.`
Until you can switch all usage to the new name internally. And then send out a PR to clean up the workaround.